apps/documentation/src/routes/documentation/tutorial/development-setup.mdx

```diff
@@ -45,20 +45,20 @@ The project will have the following structure:
 `public/`: put here all the files you want to be public
 
 `src/routes/`: All the files in this folder are considered routes. All the routes are server side rendered out of the box.
-To add server side capabilities just create a rust file with the same name as the route (i.e. `about.tsx` → `about.rs`).
+To add server side capabilities, just create a Rust file with the same name as the route (i.e. `about.tsx` → `about.rs`).
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Possible spelling mistake found. (MORFOLOGIK_RULE_EN_US)
Suggestions: `TSX`, `tax`, `CSX`, `TSA`, `TeX`, `TSS`, `ts`, `RSX`, `TWX`, `tsp`, `tux`, `ASX`, `BSX`, `JSX`, `LSX`, `OSX`, `PSX`, `TGX`, `THX`, `TMX`, `TPX`, `TRX`, `TS`, `TSB`, `TSC`, `TSD`, `TSE`, `TSG`, `TSH`, `TSI`, `TSK`, `TSL`, `TSM`, `TSN`, `TSO`, `TSP`, `TSR`, `TSV`, `TTX`, `TX`, `Tex`, `ts x`, `the`, `to`, `is`, `as`, `two`, `its`, `so`, `too`, `us`, `use`, `six`, `top`, `St`, `ten`, `text`, `box`, `ex`, `task`, `test`, `tho`, `try`, `Tom`, `ask`, `mix`, `tag`, `tea`, `tie`, `tip`, `ton`, `vs`, `Max`, `NSW`, `ash`, `fix`, `taxi`, `tin`, `toy`, `CSS`, `LSU`, `SD`, `SF`, `SG`, `SK`, `SL`, `SSR`, `TNA`, `TNT`, `TT`, `Ted`, `USA`, `USB`, `fox`, `tan`, `tap`, `taxa`, `toe`, `toss`, `tow`, `wax`, `BSA`, `BSD`, `CSA`, `CSI`, `CSU`, `DSM`, `DSO`, `DSP`, `DX`, `ESA`, `ESP`, `ISP`, `LSD`, `LST`, `MSC`, `MSN`, `NSA`, `NTSC`, `PS2`, `PSA`, `PSP`, `PSV`, `RSA`, `RSS`, `RTS`, `Rex`, `TCP`, `TLC`, `TTC`, `TV`, `aux`, `fax`, `max`, `ox`, `psi`, `sax`, `ta`, `tab`, `tar`, `thy`, `tsar`, `tub`, `tug`, `AMX`, `ASB`, `ASD`, `CSC`, `CSF`, `CSL`, `CSP`, `CSR`, `CST`, `DSC`, `DSL`, `DSS`, `DST`, `DTS`, `ESC`, `ESL`, `FSA`, `FSU`, `FTSE`, `JSA`, `MSF`, `MST`, `MTS`, `MX`, `NSF`, `NTSB`, `OSI`, `OSU`, `PSD`, `PSL`, `PSU`, `PTSD`, `RSC`, `SSA`, `SSE`, `SSL`, `SSN`, `SSP`, `SST`, `TAC`, `TCS`, `TCU`, `TDI`, `TFS`, `THC`, `TLS`, `TTL`, `TVs`, `TWA`, `Tao`, `Tess`, `USF`, `USO`, `USP`, `VAX`, `WS`, `WSU`, `hex`, `lax`, `ms`, `msg`, `sh`, `st`, `tau`, `tee`, `til`, `ASN`, `CSD`, `CSG`, `CSO`, `CSV`, `DSA`, `DSB`, `DSE`, `ESB`, `ESD`, `ESO`, `ESR`, `ESS`, `Esq`, `FSH`, `FSI`, `FSK`, `FSM`, `HSE`, `HSL`, `HSP`, `HSS`, `JSC`, `KSC`, `LSA`, `LSC`, `LSI`, `LSM`, `LSO`, `LSP`, `MMX`, `MSI`, `MSW`, `OSB`, `OSF`, `PBX`, `PSB`, `PSE`, `PSF`, `PSG`, `PSM`, `PSO`, `PSS`, `PST`, `RSI`, `RSM`, `RSV`, `SFX`, `SSM`, `SSW`, `Styx`, `TBI`, `TCA`, `TCM`, `TCR`, `TLD`, `TMZ`, `TNG`, `TOC`, `TPA`, `TPC`, `TPM`, `TRC`, `TUC`, `Tia`, `Tod`, `asp`, `ax`, `cox`, `esp`, `lux`, `pox`, `pts`, `sf`, `tai`, `tam`, `tat`, `tot`, `tusk`, `DSD`, `DSR`, `ESE`, `HSA`, `KSS`, `LSB`, `LTS`, `MDX`, `MOX`, `Mex`, `PX`, `RSP`, `SSO`, `TBA`, `TBC`, `TDD`, `TLR`, `TTA`, `USG`, `UTSA`, `WSW`, `pax`, `tel`, `tum`, `CSH`, `ESN`, `Rx`, `TCO`, `TEF`, `TPL`, `TPN`, `TTF`, `Tut`, `Twp`, `WSJ`, `nix`, `ssh`, `sty`, `tad`, `tut`, `LyX`, `TBD`, `TESL`, `Tass`, `Tue`, `lox`, `ted`, `vex`, `tbsp`, `tosh`, `tush`, `PS`, `TD`, `Tim`, `USC`, `USS`, `Hts`, `1st`, `AAX`, `ABX`, `ACX`, `ADX`, `AEX`, `AFX`, `AGX`, `AHX`, `AIX`, `AKX`, `ALX`, `ANX`, `AOX`, `APX`, `ARX`, `ASA`, `ASC`, `ASE`, `ASF`, `ASG`, `ASH`, `ASJ`, `ASK`, `ASL`, `ASM`, `ASP`, `ASQ`, `ASR`, `ASS`, `AST`, `ASU`, `ASV`, `ASW`, `ASY`, `ASZ`, `ATS`, `ATX`, `AUX`, `AVX`, `AX`, `AXX`, `As`, `Ats`, `BAX`, `BBX`, `BCX`, `BDX`, `BEX`, `BFX`, `BGX`, `BHX`, `BIX`, `BJX`, `BKX`, `BLX`, `BMX`, `BOX`, `BRX`, `BS`, `BSB`, `BSC`, `BSE`, `BSF`, `BSG`, `BSH`, `BSI`, `BSK`, `BSL`, `BSM`, `BSN`, `BSO`, `BSP`, `BSQ`, `BSR`, `BSS`, `BST`, `BSU`, `BSV`, `BSW`, `BSc`, `BTS`, `BTX`, `BUX`, `BVX`, `BYX`, `CBX`, `CCX`, `CHX`, `CLX`, `CMX`, `CNX`, `CQX`, `CS`, `CSE`, `CSM`, `CTSA`, `CTSO`, `CTs`, `CVX`, `CZX`, `Cox`, `Cs`, `DAX`, `DMX`, `DS`, `DSF`, `DSG`, `DSI`, `DSK`, `DSN`, `DSQ`, `DSU`, `DSV`, `Dix`, `EAX`, `EBX`, `EDX`, `ES`, `ES5`, `ES6`, `ESF`, `ESG`, `ESH`, `ESI`, `ESM`, `EST`, `ESV`, `ESXi`, `ESY`, `ETS`, `EX`, `Es`, `Esc`, `Etsy`, `FAX`, `FS`, `FSB`, `FSC`, `FSD`, `FSE`, `FSF`, `FSG`, `FSN`, `FSP`, `FSR`, `FSS`, `FST`, `FSW`, `FTX`, `FX`, `Fox`, `GFX`, `GPX`, `GSA`, `GSC`, `GSE`, `GSF`, `GSH`, `GSI`, `GSK`, `GSL`, `GSM`, `GSO`, `GSP`, `GSR`, `GST`, `GSW`, `GTS`, `GTX`, `HIX`, `HS`, `HSC`, `HSF`, `HSM`, `HST`, `HSV`, `HTS`, `IS`, `ISA`, `ISB`, `ISC`, `ISD`, `ISF`, `ISG`, `ISI`, `ISK`, `ISL`, `ISM`, `ISN`, `ISO`, `ISR`, `ISS`, `IST`, `ISU`, `ISV`, `ITS`, `ITSM`, `JPX`, `JS`, `JS2`, `JSB`, `JSF`, `JSL`, `JSP`, `JSU`, `KNX`, `KS`, `KSA`, `KSM`, `KSR`, `KTS`, `Ks`, `LFX`, `LIX`, `LOX`, `LS`, `LS1`, `LS2`, `LS3`, `LS4`, `LSE`, `LSF`, `LSJ`, `LSL`, `LSN`, `LSQ`, `LSW`, `LX`, `MCX`, `MPX`, `MS`, `MSA`, `MSB`, `MSD`, `MSE`, `MSG`, `MSH`, `MSK`, `MSM`, `MSO`, `MSR`, `MSS`, `MSU`, `MSc`, `MTX`, `MXX`, `Ms`, `NOX`, `NS`, `NSB`, `NSC`, `NSM`, `NSS`, `NST`, `NSU`, `NTS`, `NYX`, `OCX`, `OFX`, `OS`, `OSA`, `OSC`, `OSD`, `OSG`, `OSL`, `OSM`, `OSQ`, `OSR`, `OSS`, `OST`, `OSV`, `Os`, `PDX`, `PHX`, `PIX`, `PLX`, `PS1`, `PSC`, `PSK`, `PSN`, `PSQ`, `PSR`, `PTSI`, `QBX`, `QIX`, `QJX`, `QSE`, `QSR`, `QSV`, `REX`, `RIX`, `RPX`, `RS`, `RS7`, `RSE`, `RSF`, `RSH`, `RSO`, `RSR`, `RST`, `RSU`, `RSW`, `RTX`, `RTs`, `S`, `SA`, `SAX`, `SB`, `SC`, `SE`, `SEX`, `SGX`, `SH`, `SI`, `SJ`, `SM`, `SN`, `SO`, `SOX`, `SP`, `SRX`, `SS`, `SS7`, `SSB`, `SSD`, `SSF`, `SSG`, `SSH`, `SSI`, `SSK`, `SSS`, `SSV`, `ST`, `STS`, `STX`, `SU`, `SV`, `SW`, `SXB`, `SXM`, `SY`, `SYX`, `SZ`, `Sb`, `Sc`, `Se`, `Si`, `Sm`, `Sn`, `Sp`, `Sq`, `Sr`, `T`, `T&C`, `T&S`, `T10`, `T11`, `T12`, `T13`, `T14`, `T15`, `T16`, `T17`, `T18`, `T19`, `T20`, `T21`, `T28`, `T30`, `T37`, `T38`, `T40`, `T50`, `T7J`, `T99`, `TA`, `TAA`, `TAD`, `TAE`, `TAF`, `TAG`, `TAH`, `TAI`, `TAJ`, `TAM`, `TAN`, `TAO`, `TAP`, `TASS`, `TAT`, `TB`, `TBF`, `TBH`, `TBJ`, `TBK`, `TBL`, `TBM`, `TBP`, `TBS`, `TBT`, `TBW`, `TC`, `TCC`, `TCD`, `TCF`, `TCG`, `TCH`, `TCI`, `TCK`, `TCL`, `TCN`, `TCV`, `TCW`, `TDA`, `TDC`, `TDE`, `TDF`, `TDG`, `TDM`, `TDP`, `TDR`, `TDS`, `TDSB`, `TDT`, `TEB`, `TEC`, `TED`, `TEDx`, `TEE`, `TEI`, `TEM`, `TEP`, `TEQ`, `TER`, `TES`, `TESS`, `TEU`, `TEV`, `TF`, `TFA`, `TFB`, `TFC`, `TFE`, `TFF`, `TFK`, `TFL`, `TFR`, `TFT`, `TG`, `TGA`, `TGB`, `TGD`, `TGE`, `TGF`, `TGI`, `TGL`, `TGN`, `TGP`, `TGS`, `TGU`, `TGV`, `TGW`, `TGZ`, `TH`, `THA`, `THB`, `THD`, `THF`, `THM`, `THP`, `THS`, `THT`, `TI`, `TIC`, `TID`, `TIG`, `TIP`, `TIS`, `TIV`, `TJ`, `TJC`, `TJJ`, `TJK`, `TJS`, `TK`, `TKD`, `TKK`, `TKL`, `TKM`, `TKO`, `TKP`, `TKS`, `TKT`, `TL`, `TLA`, `TLB`, `TLE`, `TLF`, `TLH`, `TLM`, `TLN`, `TLP`, `TLT`, `TLV`, `TM`, `TMA`, `TMB`, `TMC`, `TMD`, `TME`, `TMG`, `TMI`, `TMJ`, `TMN`, `TMP`, `TMS`, `TMT`, `TMV`, `TN`, `TNB`, `TNC`, `TND`, `TNE`, `TNF`, `TNK`, `TNL`, `TNM`, `TNN`, `TNO`, `TNP`, `TNR`, `TNS`, `TNW`, `TO`, `TOB`, `TOE`, `TOF`, `TOG`, `TOK`, `TOM`, `TOP`, `TOR`, `TOS`, `TOT`, `TP`, `TPE`, `TPF`, `TPG`, `TPI`, `TPJ`, `TPO`, `TPP`, `TPR`, `TPS`, `TPT`, `TPV`, `TQ`, `TQC`, `TQM`, `TQS`, `TR`, `TRE`, `TRI`, `TRL`, `TRM`, `TRP`, `TRS`, `TRT`, `TRY`, `TSAF`, `TSIC`, `TSLA`, `TSMA`, `TSVP`, `TTD`, `TTG`, `TTI`, `TTO`, `TTP`, `TTR`, `TTS`, `TTT`, `TTU`, `TTY`, `TTs`, `TU`, `TUA`, `TUB`, `TUG`, `TUI`, `TUN`, `TUP`, `TUV`, `TV7`, `TV8`, `TVA`, `TVH`, `TVM`, `TVO`, `TVP`, `TVQ`, `TVS`, `TVU`, `TW`, `TWC`, `TWD`, `TWE`, `TWF`, `TWS`, `TWU`, `TWV`, `TXK`, `TXL`, `TXT`, `TY`, `TYC`, `TYO`, `TZ`, `TZA`, `TZC`, `TZN`, `TZS`, `Ta`, `Tad`, `Tai`, `Tb`, `Tc`, `Te`, `Tet`, `Th`, `Thu`, `Ti`, `TiB`, `Tl`, `Tm`, `ToC`, `ToS`, `Tsai`, `Tu`, `Twix`, `Ty`, `UBX`, `UGX`, `ULX`, `US`, `USD`, `USE`, `USI`, `USJ`, `USN`, `USR`, `UX`, `VDX`, `VFX`, `VSB`, `VSC`, `VSD`, `VSF`, `VSG`, `VSL`, `VSM`, `VSN`, `VSP`, `VSR`, `VSS`, `VST`, `VSV`, `Vox`, `WSA`, `WSB`, `WSC`, `WSD`, `WSF`, `WSL`, `WSM`, `WSN`, `WSP`, `WSS`, `WST`, `Wix`, `X`, `XS`, `XSN`, `XSS`, `XXX`, `ZSE`, `ZSL`, `ZSP`, `ZZX`, `Zs`, `ass`, `bx`, `bxs`, `cs`, `cts`, `cx`, `dox`, `est`, `gs`, `isl`, `ism`, `isn`, `ix`, `ks`, `lix`, `ls`, `pix`, `pyx`, `qts`, `rs`, `s`, `sex`, `sq`, `std`, `t`, `tase`, `tb`, `tbs`, `ti`, `tic`, `tit`, `tmp`, `tn`, `tog`, `tom`, `tor`, `tr`, `tun`, `twp`, `usu`, `x`, `xix`, `xx`, `xxx`, `DEX`, `PS3`, `PS4`, `PS5`, `TIF`, `TPU`, `TSMC`, `XLSX`, `XSD`, `XSL`, `tmux`
Rule: https://community.languagetool.org/rule/show/MORFOLOGIK_RULE_EN_US?lang=en-US
Category: TYPOS
<!-- __reviewdog__:ChA0YjUwOTBkMmU5NzE4NTJkEgxMYW5ndWFnZVRvb2w= -->